Zend/tests/heredoc_nowdoc/heredoc_print.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Dump heredoc tags
+--INI--
+zend.assertions=1
+--FILE--
+<?php
+try {
+    assert(false &&
+    <<<HTML
+    val inside
+    HTML
```

Comment:
What would happen if:

- The string contains a line `HEREDOC`?
- The string contains special characters that would be escaped by `zend_ast_export_str()`?

Is it guaranteed that the output is valid PHP in all cases?